featuretools/tests/conftest.py

```diff
@@ -21,14 +19,24 @@
 
 @pytest.fixture()
 def dask_cluster():
-    with LocalCluster() as cluster:
-        yield cluster
+    distributed = pytest.importorskip(
+        "distributed",
+        reason="Dask not installed, skipping",
```

Comment:
Would it be a good idea to have a Dask error message saved instead of copying it for all tests?

Reply:
We could do that, but the only people that ever see this message are developers, so I don't think it's critical that this message be the same everywhere.

Reply:
Makes sense. I was wondering more from a code cleanliness perspective.